addons/website_sale/models/res_company.py

```diff
@@ -9,9 +9,19 @@ class ResCompany(models.Model):
 
     website_sale_onboarding_payment_acquirer_state = fields.Selection([('not_done', "Not done"), ('just_done', "Just done"), ('done', "Done")], string="State of the website sale onboarding payment acquirer step", default='not_done')
 
+    @api.model
+    def is_odoo_payment_available(self):
+        payments_methods = [m for m, _ in self.env["payment.acquirer.onboarding.wizard"]._selection_payment_methods()]
+        return "odoo" in payments_methods
```

Comment:
Honestly, I'd prefer a duplicate check of  `self.env.company.country_id.code in ODOO_PAYMENTS_DEPLOYED_COUNTRIES` than an obscure check in the available selections of a field.

Reply:
Yes, you're right.
So, I moved `ODOO_PAYMENTS_DEPLOYED_COUNTRIES`and `is_odoo_payment_available()` is `payments/models/res_company.py`and used `is_odoo_payment_available()` everywhere we checked with `ODOO_PAYMENTS_DEPLOYED_COUNTRIES` to avoid duplicates ;)